packages/fast-components-react-base/src/index.spec.ts

```diff
@@ -0,0 +1,14 @@
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure } from "enzyme";
+import * as path from "path";
+import { includesAllSubdirectoriesAsNamedExports } from "../../../build/helpers/file-includes-all-subdirectories-as-named-exports";
+
+configure({ adapter: new Adapter() });
+
+describe("index.ts", (): void => {
+    test("should export all components in the src directory", () => {
+        expect(
+            includesAllSubdirectoriesAsNamedExports(path.resolve(__dirname, "index.ts"))
+        ).toEqual([]);
```

Comment:
This is interesting from a name standpoint as I expect a boolean value to be returned when I look at the function.

Does this "includesAllSubdirectoriesAsNamedExports" - Yes. Instead I see, "includesAllSubdirectoriesAsNamedExports" - empty array.

Reply:
A boolean would only tell you that it failed, but not which exports are missing. @janechu suggested the function name, but I am open to changing it so the expectation is a listing of missing exports or an empty array.

Reply:
you could use true when the array is empty and give an actual error when it returns with the appropriate details, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error

Reply:
Then you could simply expect the function to not throw

Reply:
I'll look into implementing as described above.